web/modules/custom/demo_content/demo_content.info.yml

```diff
@@ -0,0 +1,5 @@
+name: Demo content
+type: module
+description: Demo content for Joinup project
+core: 8.x
+package: joinup
```

Comment:
Joinup with uppercase, or it will be shown in a separate group (this happens on my local).